### DISCUSSION TITLE
[feature] Appkit (Specifically social logins) functionality in PWA / standalone mode on iOS

### DISCUSSION THREAD

**Discussion by R-qanawati** [reown-com/appkit#3702] [Ideas]
### What problem does this new feature solve?

When using AppKit's social authentication (Google/Apple/X) in PWA/standalone mode on iOS, the auth popup results in about:blank or fails to complete the OAuth flow. This occurs specifically when the app is installed as a PWA and launched from the home screen.

### Describe the solution you'd like

1. Open auth successfully
2. Complete OAuth flow
3. Return to PWA with authenticated session

**Comment by R-qanawati** [comment]
When i have my app running in PWA i get errors such as 

`Unable to post message to https://auth.magic.link. Recipient has origin https://secure.walletconnect.org.`
`[Error] Origin https://<myvercellink>.vercel.app not found on Allowlist - update configuration to enable social login`

my vercel.app is an authorized origin on reown dashboard

**Comment by julher625** [comment]
See this link with the documentation:
https://docs.reown.com/cloud/verify

**Comment by R-qanawati** [comment]
> See this link with the documentation: https://docs.reown.com/cloud/verify

I verified the domain but the issue still persists

**Comment by arein** [comment]
Do you have a link where we can reproduce?
